src/components/10-atoms/icon/index.scss

```diff
@@ -1,4 +1,15 @@
-:host(:not([size=auto])) svg {
+:host([size='small']) svg {
+  width: 20px;
+  height: 20px;
+}
+
+:host(:not([size])) svg,
+:host([size='medium']) svg {
```

Comment:
This line could be changed to:

`:host([size]) svg {`

so we would not need an explicit size "medium" at all.

Reply:
Advantage of that is: If a user writes `<axa-icon size="hello"`, it would still fallback to the default value. also if `size=""` is being set.